modules/bitgo/test/v2/unit/coins/utxo/recovery/backupKeyRecovery.ts

```diff
@@ -203,32 +237,38 @@ function run(
 
     if (params.hasUserSignature && params.hasBackupSignature) {
       it('has no placeholder signatures', function () {
```

Comment:
```suggestion
      it('has no placeholder signatures', function (this: mocha.Context) {
```
then you can get rid of the `@ts-expect-error` further down